spec/generators/paper_trail/install_generator_spec.rb

```diff
@@ -124,4 +125,50 @@
       )
     end
   end
+
+  describe "`--json` option set to `true`" do
+    before do
+      prepare_destination
+      run_generator %w[--json]
+    end
+
+    it "generates a migration for creating the 'versions' table with `object` type json" do
+      expect(destination_root).to(
+        have_structure {
+          directory("db") {
+            directory("migrate") {
+              migration("create_versions") {
+                contains "t.json     :object, limit: TEXT_BYTES"
+              }
+            }
+          }
+        }
+      )
+    end
+  end
+
+  describe "`--with-changes` and `--json` option set to `true`" do
+    before do
+      prepare_destination
+      run_generator %w[--with-changes --json]
+    end
+
+    # rubocop:disable Layout/LineLength
```

Comment:
Please do not disable rubocop. In this case, please shorten the test title. Feel free to add comments above the test if necessary.